config/environments/development.rb

```diff
@@ -18,6 +18,11 @@
   config.action_mailer.default_url_options = {
     host: (ENV["MAIL_LINK_HOST"] || "localhost:3000")
   }
+  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.smtp_settings = {
+    address: ENV['MAIL_HOST'] || 'localhost',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.